sycl/plugins/opencl/CMakeLists.txt

```diff
@@ -9,9 +9,16 @@
 #TODO: remove dependency on pi.hpp in sycl project.
 #TODO: Currently, the pi.hpp header is common between sycl and plugin library sources. 
 #This can be changed by copying the pi.hpp file in the plugins project.
+configure_file(
+  ${CMAKE_CURRENT_SOURCE_DIR}/../../source/version.rc.in
+  ${CMAKE_CURRENT_BINARY_DIR}/version.rc
+  @ONLY)
+
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
 add_library(pi_opencl SHARED
   "${sycl_inc_dir}/CL/sycl/detail/pi.h"
   "pi_opencl.cpp"
+  ${CMAKE_CURRENT_BINARY_DIR}/version.rc
```

Comment:
Here

Reply:
GitHub dropped my second note, but it was on line 64 of the same file.

Reply:
We should *not* be doing that. The plugins version control happens (will happen) at run-time with dynamic queries to the plugin and matching against what SYCL RT can work with.

Reply:
Sorry, this line should have been removed.
Due to the local "check.py" complain about ABI policy failures, I thought I had to implement a versioning of libpi_opencl.so just like libsycl.so did.
Obviously, that's not right.
I will remove this.